docs/content/contributing/_index.md

```diff
@@ -6,6 +6,10 @@ title = "Contribution Guidelines"
   parent="contributing"
 +++
 
+### Code Style
+
+When contributing code, in addition to following the [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines), please follow the same [code style](https://github.com/mongodb/mongo/wiki/Server-Code-Style), [design guidelines](https://github.com/mongodb/mongo/wiki/Server-Design-Guidelines), and [style guidelines](https://github.com/mongodb/mongo/wiki/Style-Guidelines) as [mongodb/mongo](https://github.com/mongodb/mongo). Additions and exceptions are listed below.
```

Comment:
We may need to remove https://github.com/mongodb/mongo/wiki/Server-Code-Style. Our .clang-format differs from the one in mongodb/mongo. Since we have a lint task that enforces style, we might not need to mention it.

Reply:
Yes, what if we just mention that users should clang-format their code?  Maybe we talk about that somewhere else? 
 Also, in the past we've taken a lot of cues from the Google style guide: https://google.github.io/styleguide/cppguide.html

Reply:
Done